src/events/tests/conftest.py

```diff
@@ -1,18 +1,37 @@
 import pytest
+import pytz
+
+from django.utils.dateparse import parse_datetime
+from django.utils.timezone import is_naive
 
 from events.models import Location, SponsoredEvent, Time
 
 
+cst = pytz.timezone('Asia/Taipei')
+
+
+@pytest.fixture
+def get_time(day):
```

Comment:
Have hint in function name or function doc or comment on why it's needed or what is its purpose?